root/report/instruments_without_image.tt

```diff
@@ -0,0 +1,42 @@
+[%- WRAPPER 'layout.tt' title=l('Instruments without image') full_width=1 -%]
```

Comment:
Fixed in https://github.com/metabrainz/musicbrainz-server/pull/444/commits/cf109ca175a52f75b7cc016a2381eca2ab8d943f